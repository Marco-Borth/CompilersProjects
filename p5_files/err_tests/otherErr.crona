a:int;
b:bool;
arr : int array[10];
fn:void(){
	write arr;
	write c;
}

newFunc : bool() {
	return 10;
}
