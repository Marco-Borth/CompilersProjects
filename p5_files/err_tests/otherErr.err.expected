FATAL [6,8]: Undeclared identifier
Type Analysis Failed
